ios/StripeTerminalReactNative.swift

```diff
@@ -989,6 +989,38 @@ class StripeTerminalReactNative: RCTEventEmitter, DiscoveryDelegate, BluetoothRe
         self.collectRefundPaymentMethodCancelable = nil
     }
 
+    @objc(collectData:resolver:rejecter:)
+    func collectData(params: NSDictionary, resolver resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) {
+        let collectDataTypeParam = params["collectDataType"] as? String ?? ""
+        let enableCustomerCancellation = params["enableCustomerCancellation"] as? Bool ?? false
+
+        let collectDataType = Mappers.mapToCollectDataType(collectDataTypeParam)
+        guard let collectDataType else {
+            resolve(Errors.createError(code: CommonErrorType.InvalidRequiredParameter, message: "You must provide correct collectDataType parameter."))
+            return
+        }
+
+        let collectDataConfig: CollectDataConfiguration
+        do {
+            collectDataConfig = try CollectDataConfigurationBuilder().setCollectDataType(collectDataType)
```

Comment:
`enableCustomerCancellation` is missing here

Reply:
`enableCustomerCancellation` seem have not support in IOS SDK yet

Reply:
oh! thanks for pointing that out 👍